maven/pom.xml

```diff
@@ -124,11 +124,6 @@
 				<version>${project.version}</version>
 			</dependency>
 			<dependency>
-				<groupId>biz.aQute.bnd</groupId>
```

Comment:
OK, for some reason I thought #1794 removed the reference, but it was just in the project pom. So this is fine.